modules/ossm-release-2-6-2.adoc

```diff
@@ -0,0 +1,46 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-2_{context}"]
+= {SMProductName} version 2.6.2
+
+This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.2, and includes the following ServiceMeshControlPlane resource version updates: 2.6.2, 2.5.5 and 2.4.11.
```

Comment:
Looks like this one didn't go through (and now I have accidentally deleted the original comment :sweat_smile:)
```suggestion
This release of {SMProductName} updates the {SMProductName} Operator version to 2.6.2, and includes the following `ServiceMeshControlPlane` resource version updates: 2.6.2, 2.5.5 and 2.4.11.
```

Reply:
otherwise LGTM